docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -265,6 +266,26 @@ With this configuration, all events with `log_type: normal` are sent to an
 index named +normal-{version}-{localdate}+, and all events with
 `log_type: critical` are sent to an index named
 +critical-{version}-{localdate}+.
+endif::apm-server[]
+
+ifdef::apm-server[]
+You can set the index dynamically by using a format string to access any event
+field. For example, this configuration uses the field, `processor.event`,
+to set the index:
+
+["source","yaml",subs="attributes"]
+------------------------------------------------------------------------------
+output.elasticsearch:
+  hosts: ["http://localhost:9200"]
+  index: "apm-%\{[observer.version]\}-%\{[processor.event]\}-%\{+yyyy.MM.dd}\" <1>
```

Comment:
i'd note `observer` is the apm-server, for people not familiar with ECS

Reply:
This is a good point. I'll note that in the `<1>` callout